### ISSUE TITLE
Set up a target directory for the command line usage?

### ISSUE THREAD

**Issue by xwshiba** [bgeron/diff-pdf-visually#11]
Hi, thanks for the great tool! 
I'm wondering if it is possible for users to set up a target directory to save the diff pngs for the command line usage?
Currently it will automatically save to the /tmp/diffpdf* location. And since the tmp location is added to the function according to this closed issue: https://github.com/bgeron/diff-pdf-visually/issues/6, It would be helpful if we can set our own from the command line.
I think the relevant part is line 70 to 74 in __main__.py.

**Comment by infotexture** [comment]
@bgeron I ran into this too. I tried passing `--tempdir` to the CLI, but got an error:

```
diff-pdf-visually: error: unrecognized arguments: --tempdir /path/to/folder
```

Could it be that this option is only supported by the Python function, but not the CLI?

**Comment by bgeron** [comment]
Thanks both for the report, and for @ropeladder for providing a fix! With a tweak, it's now released in v1.8.0 with `--outdir`.

(And apologies for my late response! Sometimes open source slips through the cracks.)

I think this resolves the issue, so closing, but let me know if this doesn't solve your use case.

**Comment by ropeladder** [comment]
Looks great! `--outdir` is definitely clearer.
